src/domain/banner/bannerConstants.ts

```diff
@@ -0,0 +1 @@
+export const BANNER_SOME_IMAGE = '/images/activities_SoMe-share.jpg';
```

Comment:
naming of this file is not in line with other files with constant values. Shoul we also name this "constants.ts"?